src/hooks/core/push-notification/usePushNotificationHook.ts

```diff
@@ -8,6 +8,7 @@ import {useRecoilState} from 'recoil';
 
 import ChannelList from '../../../database/schema/ChannelListSchema';
 import ChatSchema from '../../../database/schema/ChatSchema';
+import currentChatScreenAtom from '../../../atom/currentChatScreenAtom';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `currentChatScreenAtom` is a new addition to the code. Ensure that this atom is correctly defined and exported in the specified path. Also, make sure it's being used appropriately throughout the code.

```diff
+ import currentChatScreenAtom from '../../../atom/currentChatScreenAtom';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->